src/dymaptic.GeoBlazor.Core/Components/Widgets/BasemapLayerListWidget.cs

```diff
@@ -77,9 +106,11 @@ public class BasemapLayerListWidget : Widget
     ///     Returns the modified base <see cref="ListItem" />
     /// </returns>
     [JSInvokable]
-    public Task<ListItem>? OnBaseListItemCreated(ListItem item)
+    public async Task<object> OnBaseListItemCreated(ListItem item)
```

Comment:
just curious, why `ListItem` was changed to a more general `object`?

Reply:
Good question, and I had to revisit to remember why. The return value of a `JSInvokable` is going back to JS, and we have issues sometimes with serialization when not casting to object first. You'll see a few other places in GeoBlazor where we cast to object before sending to JS. I will add a comment here.